src/Raven.Server/Smuggler/Documents/DatabaseSmuggler.cs

```diff
@@ -637,6 +638,10 @@ await using (var actions = _destination.DatabaseRecord())
                 {
                     await actions.WriteDatabaseRecordAsync(databaseRecord, result, _options.AuthorizationStatus, _options.OperateOnDatabaseRecordTypes);
                 }
+                catch (LicenseLimitException)
+                {
```

Comment:
what would happened to the database in such case? would we be able to delete it?

Reply:
The database will be empty, and you will be able to delete it